src/file_system/mod.rs

```diff
@@ -1152,4 +1169,43 @@ pub mod tests {
 
         assert_eq!(root, expected_root)
     }
+
+    fn path_from_string_vector(names: &Vec<&str>) -> Path {
+        let labels: Vec<Label> = names.iter().map(|n| Label::from(*n)).collect();
+        Path::from(labels)
+    }
+    
+    #[test]
+    fn build_path_from_vector() {
+        let names = vec!["a", "b", "c"];
```

Comment:
If I recall correctly other places in the repo prefer `bar`, `baz`, etc. for this kind of test enumeration.

Reply:
This is true, I have been using those simplistic names, but don't really have guidelines :sweat_smile: Consistency makes sense though :ok_hand: